source/flux/parts/settings.js

```diff
@@ -35,6 +39,9 @@ const CREDENTIALS_VALIDATE_FAILURE = 'settings/CREDENTIALS_VALIDATE_FAILURE'
 const SET_FEEDBACK = 'settings/SET_FEEDBACK'
 const CHANGE_THEME = 'settings/CHANGE_THEME'
 const SIS_ALERT_SEEN = 'settings/SIS_ALERT_SEEN'
+const SET_TOUCHID_STATUS = 'settings/SET_TOUCHID_STATUS'
+const LOAD_TOUCHID_STATUS = 'settings/LOAD_TOUCHID_STATUS'
```

Comment:
I don't think we need to store the Touch ID status in Redux.

Reply:
If not, how will the app remember whether or not the user wants this feature enabled or not?

Reply:
I think @hannesmcman is right. We need some way to give the user the option to opt-in or out.